### DISCUSSION TITLE
cn/2024/12/11/cognitive-driven/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#196] [Announcements]
# cn/2024/12/11/cognitive-driven/

https://hongtaoh.com/cn/2024/12/11/cognitive-driven/

**Comment by Liam-Zhong** [comment]
谢谢博主分享。其实开卷有得，也不一定是书本带来的东西，自己的思考也是心得嘛。

在我这里很多道理人都知道，但是做不到。所以比较偏爱小说🤣比如最近在看的金瓶梅

**Comment by wangyunzi** [comment]
我学习英语就是乱学，导致现在还没啥进步，但是多听新闻感觉听力进步了哈哈哈
